typescript/sudoAsUser.ts

```diff
@@ -61,7 +61,8 @@ const matchDomain = '%@looker.com'
  * @returns {Promise<undefined | IUser>} Returns the first matched user, or undefined if no match
  */
 const anyoneButMe = async (userId: number, emailPattern: string) => {
-  const all = await sdk.ok(sdk.search_users({email: emailPattern, page: 0, per_page: 2}))
+  const all = await sdk.ok(sdk.search_users({email: emailPattern, page: 1, per_page: 2}))
+  // const all = await sdk.ok(sdk.search_users({email: emailPattern, page: 0, per_page: 2}))
```

Comment:
🤔

Reply:
oh yuck. copy pasta. thanks for catching.